src/main/java/baritone/process/elytra/BlockStateOctreeInterface.java

```diff
@@ -27,28 +28,31 @@ public final class BlockStateOctreeInterface {
 
     private final NetherPathfinderContext context;
     private final long contextPtr;
+    private final DimensionType dimType;
     transient long chunkPtr;
 
     // Guarantee that the first lookup will fetch the context by setting MAX_VALUE
     private int prevChunkX = Integer.MAX_VALUE;
     private int prevChunkZ = Integer.MAX_VALUE;
 
-    public BlockStateOctreeInterface(final NetherPathfinderContext context) {
+    public BlockStateOctreeInterface(final NetherPathfinderContext context, final DimensionType dimType) {
         this.context = context;
         this.contextPtr = context.context;
+        this.dimType = dimType;
```

Comment:
Could potentially take the dimension info from `context`

Reply:
NetherPathfinderContext just has the id of the dimension which the min y can't directly be taken from

Reply:
It does have the min y here https://github.com/cabaletta/baritone/blob/04813a8312550ba5c18a0611b175f5062d21db46/src/main/java/baritone/process/elytra/NetherPathfinderContext.java#L83